diff --git a/ui/components/multichain/account-overview/account-overview-tabs.tsx b/ui/components/multichain/account-overview/account-overview-tabs.tsx
index c695684..c4ae8ce 100644
--- a/ui/components/multichain/account-overview/account-overview-tabs.tsx
+++ b/ui/components/multichain/account-overview/account-overview-tabs.tsx
@@ -178,7 +178,7 @@ export const AccountOverviewTabs = ({
 
       {showPrivateBalance && (
         <Tab
-          name="Private balance"
+          name="cTokens"
           tabKey={AccountOverviewTabKey.PrivateBalance}
           data-testid="account-overview__private-balance-tab"
         >
